src/events/guildCreate.js

```diff
@@ -1,7 +1,24 @@
-const { setServerPrefix } = require("../utils/functions");
+const { setServerPrefix, getServerPrefix } = require("../utils/functions");
+const { MessageEmbed } = require("discord.js");
 module.exports = {
     name: "guildCreate",
     async execute(bot, guild) {
         setServerPrefix(guild.id, "!");
+        let channelID
+        let channels = guild.channels.cache
+        channelLoop:
+        for (let c of channels) {
+          let channelType = c[1].type
+          if (channelType === "text") {
+              channelID = c[0]
+              break channelLoop
+            }
+        }  
+        let currentPrefix = getServerPrefix(guild.id)
+        let channel = client.channels.cache.get(guild.systemChannelID || channelID);
```

Comment:
Change `client` to `bot` since client is not defined.

Reply:
i think i fixed it now